forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -5187,12 +5187,6 @@ public final boolean sharesNameWith(final Card c1) {
             } else if (getName().isEmpty()) {
                 // if this does not have a name, then there is no name to share
                 return false;
-            } else {
-                // check if this card has a name from a face
-                // in general token creatures does not have this
-                // TODO add check if face is legal in the format of the game
-                // name does need to be a non-legendary creature
-                return StaticData.instance().getCommonCards().isNonLegendaryCreatureName(getName());
```

Comment:
This one should return true

If c1 has all non legendary names, but this doesn't,
then the next line fails

Reply:
oops